Av1an/compose.py

```diff
@@ -215,9 +215,9 @@ def compose_vvc(a: Args, c: Chunk) -> List[str]:
     :return: A list of commands
     """
     yuv_file = get_yuv_file_path(c).as_posix()
-    return [
-        f'vvc_encoder -c {a.vvc_conf} -i {yuv_file} {a.video_params} -f {c.frames} --InputBitDepth=10 --OutputBitDepth=10 -b {c.output}',
-    ]
+    return (
+        ('vvc_encoder', '-c', a.vvc_conf, '-i', yuv_file, *a.video_params, '-f', c.frames, '--InputBitDepth=10', '--OutputBitDepth=10', '-b', c.output),
```

Comment:
Need an explicit string conversion.
```suggestion
        ('vvc_encoder', '-c', a.vvc_conf, '-i', yuv_file, *a.video_params, '-f', str(c.frames), '--InputBitDepth=10', '--OutputBitDepth=10', '-b', c.output),
```